api/openapi-spec/swagger.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
    "title": "Kubernetes",
-   "version": "unversioned"
+   "version": "v1.6.0"
```

Comment:
No, it should be different if we build it from different branches/tags. Basically it should be representative of the code it generate spec from. That is why I used pkg/version's version. The swagger.json file is auto-generated.